finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,45 @@
+import { isArray } from './utils';
+
+export function min() {
+  let minimo;
+  let i;
+
+  if (arguments === undefined || arguments.length === 0) {
+    return undefined;
+  }
+
+  if (!isArray(...arguments)) {
+    minimo = Math.min(...arguments);
+  } else {
+    let arr1=[];
+    for (i = 0; i < arguments.length; i++) {
+      arr1.push(...(arguments[i]));
+    }
+    minimo = Math.min(...arr1);
+  }
+  return minimo;
+}
+
+export function copy(args) {
+  let objClone;
+  let i;
+  if (!isArray(...arguments)) {
+    objClone = { ...args };
+  } else {
+    const arr1 = [];
+    for (i = 0; i < arguments.length; i++) {
+      arr1.push(...(arguments[i]));
+    }
```

Comment:
I'd extract this piece of code into a function, so we reuse it at `min`. What do you think ?

Reply:
I will reuse a function similar to this one in order to simplify the code